tests/test_tutorials.py

```diff
@@ -26,16 +26,16 @@
 #    b. Expected objective value.
 tutorials = [
     (('westeros', 'westeros_baseline'),
-     [('solve-objective-value', 207544.09375)]),
+     [('solve-objective-value', 321750.78125)]),
```

Comment:
these changes are coming in from the discount change?

Reply:
as far as I understand the issue, yes

Reply:
We identified the issue and will add a commit and future PR